packages/create-app/templates/template-next-static-export/.gitignore

```diff
@@ -10,8 +10,8 @@
 /coverage
 
 # next.js
-/.next/
-/out/
+/.next
+/out
```

Comment:
I changed this .gitignore because it wasn't being respected by the globby implementation in the sample code provided here they mention there are some bugs with it:

https://github.com/gliviu/dir-compare?tab=readme-ov-file#implement-gitignore-filter